src/features/feed/models/feed.model.ts

```diff
@@ -1,2 +1,7 @@
-// boiler plate
-import React from "react";
\ No newline at end of file
+export type FeedPostResponseModel = {
+  readonly id: number;
+  readonly media?: string;
+  readonly userId: number;
+  readonly content: string;
+  readonly categoryName: string;
```

Comment:
We might also want to include createdDate and updatedDate.

Reply:
Definitely, good catch!

Reply:
Actually if we think about it, we dont really need a created date for a feed, because it's created when the user starts this for the first time. As for updated, maybe actually as well because I actually cant think of a reason why we would need when the feed was last updated. If you can think of a reason then add it if not then what you have is fine.

Reply:
Also, this wouldn't work right now because the backend response doesn't accept createdAt or updatedAt. Think you should create a new ticket and add those fields as well if you do decide to keep it.

Reply:
I was thinking maybe we could use it to check whether the feed needs to be updated. I'm not really sure how we're going to update the feed anymore.